lib/slimdown/page.rb

```diff
@@ -67,6 +68,20 @@ def children
       Slimdown::Folder.new(folder).pages
     end
 
+    # The parent of this document
+    #
+    # @return [<Slimdown::Page>|nil] the parent, if it exists
+    def parent
+      folder = @absolute_path
+      folder.slice! '.md'
+
+      parent = Pathname.new(folder).parent.to_s + '.md'
```

Comment:
It's probably better to use `@absolute_path.chomp('.md')` instead of the mutated local variable `folder`. I see that I had done that in `#children` and `#path` too, which I will fix.

Reply:
What about `File.expand_path('..',absolute_path).concat('.md')`?